packages/console/src/pages/Applications/index.tsx

```diff
@@ -128,7 +129,12 @@ function Applications() {
         isOpen={isShowingCreationForm}
         onClose={async (newApp) => {
           if (newApp) {
-            navigate(buildGuidePathname(newApp.id), { replace: true });
+            const buildNavigatePath =
+              newApp.type === ApplicationType.MachineToMachine
+                ? buildDetailsPathname
+                : buildGuidePathname;
```

Comment:
should have some better solution. refactor later?

Reply:
We should provide guide for M2M as well. I have a task for that and will handle it later